src/renderer/screens/settings/sections/Help/ResetButton.js

```diff
@@ -3,17 +3,20 @@
 import React, { useCallback } from "react";
 import styled from "styled-components";
 import { useTranslation } from "react-i18next";
+import { useGlobalSyncState, SyncSkipUnderPriority } from "@ledgerhq/live-common/lib/bridge/react";
+import { useCountervaluesPolling } from "@ledgerhq/live-common/lib/countervalues/react";
 import logger from "~/logger";
 import { useHardReset } from "~/renderer/reset";
 import ConfirmModal from "~/renderer/modals/ConfirmModal";
 import ResetFallbackModal from "~/renderer/modals/ResetFallbackModal";
-import { SyncSkipUnderPriority } from "@ledgerhq/live-common/lib/bridge/react";
 import Box from "~/renderer/components/Box";
 import Button from "~/renderer/components/Button";
 import IconTriangleWarning from "~/renderer/icons/TriangleWarning";
 import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
 import { useActionModal } from "./logic";
 
+let reset;
```

Comment:
i'm not really sure the global variable pattern is a good idea. This button could be used at different places too.

Reply:
yes indeed i should have put it in a ref